README.adoc

```diff
@@ -69,6 +69,32 @@ kubectl create -f samples/echo-function.yaml
 kubectl logs -f [echo-pod-name] -c sidecar
 ```
 
+== Run the gRPC Sample
+
+=== Build the gRPC sample image
+```
+samples/grpc/dockerize
```

Comment:
I'd centralize the (non-maven) dockerizations in the dockerize script at the root of the project

Reply:
So far the main `dockerize` script is for required components. This one is only required for the grpc sample.